### PR TITLE
Heizi.HeiziFlashTools version 0.0.8

### PR DESCRIPTION
**Pull request by ElisaMin**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/90358)

### CODE REVIEW

File: manifests/h/Heizi/HeiziFlashTools/0.0.8/Heizi.HeiziFlashTools.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Heizi.HeiziFlashTools
+PackageVersion: 0.0.8
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://github.com/ElisaMin/Heizi-Flashing-Tools/releases/download/0.0.8/HeiziFlashTools-lzma-0.0.8.exe
+  InstallerSha256: 8FD7AE07F6B176B024DAA66F9DCC12762B5C0058C3AFF07A14A54F7DB937DB5F
```
Comment (stephengillie) [inline manifests/h/Heizi/HeiziFlashTools/0.0.8/Heizi.HeiziFlashTools.installer.yaml:10]:
```suggestion
  InstallerSha256: 8FD7AE07F6B176B024DAA66F9DCC12762B5C0058C3AFF07A14A54F7DB937DB5F
  InstallerSwitches:
    Silent: /qn
```

**Review by ElisaMin — COMMENTED** [review]
thx

**Comment by ElisaMin** [comment]
@microsoft-github-policy-service agree

File: manifests/h/Heizi/HeiziFlashTools/0.0.8/Heizi.HeiziFlashTools.locale.zh-CN.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Heizi.HeiziFlashTools
+PackageVersion: 0.0.8
+PackageLocale: zh-CN
+Publisher: Heizi
+PackageName: HeiziFlashTools
+License: ' GPL-3.0 License'
```
Comment (mdanish-kh) [inline manifests/h/Heizi/HeiziFlashTools/0.0.8/Heizi.HeiziFlashTools.locale.zh-CN.yaml:9]:
```suggestion
License: GPL-3.0
LicenseUrl: https://github.com/ElisaMin/Heizi-Flashing-Tools/blob/main/LICENSE
```
* Quotes are not needed
* Added License Url

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**